cvmfs/cache_posix.cc

```diff
@@ -269,9 +269,9 @@ PosixCacheManager *PosixCacheManager::Create(
     const string &cache_path,
     const bool alien_cache,
     const RenameWorkarounds rename_workaround,
-    const bool do_refcount) {
+    const bool do_refcount, const bool cleanup_unused_first) {
   UniquePtr<PosixCacheManager> cache_manager(
-      new PosixCacheManager(cache_path, alien_cache, do_refcount));
+      new PosixCacheManager(cache_path, alien_cache, do_refcount,cleanup_unused_first));
```

Comment:
```suggestion
      new PosixCacheManager(cache_path, alien_cache, do_refcount, cleanup_unused_first));
```
Whitespace. Can you rerun clang-fmt on changed files?